centreon/tests/e2e/features/Knowledges/common.ts

```diff
@@ -0,0 +1,35 @@
+const configureKB = (url, account, password): void => {
+  // Visit the Configuration of 'Knowledge Base' page
+  cy.visit('/centreon/main.php?p=50133&o=knowledgeBase')
+  cy.wait('@getTimeZone');
+  // Wait until the 'knowledge base url' is visible in the DOM
+  cy.waitForElementInIframe('#main-content', 'input[name="kb_wiki_url"]');
+  // Type a value in the 'Knowledge base url'
+  cy.getIframeBody().find('input[name="kb_wiki_url"]').clear().type(url);
+  // Type a value in the 'Knowledge base account'
+  cy.getIframeBody().find('input[name="kb_wiki_account"]').clear().type(account);
+  // Type a value in the 'knowledge base password'
+  cy.getIframeBody().find('input[name="kb_wiki_password"]').clear().type(password);
+  // Click on the 'Save' button
+  cy.getIframeBody().find('input[value="Save"]').click();
+  //cy.wait('@getTimeZone');
+  cy.exportConfig();
+};
+
+const getMediaWikiContainerPort = () => {
+  return cy.exec('docker ps --format "{{.Names}}: {{.Ports}}"').then((result) => {
```

Comment:
why not using `getMappedPort` function ? https://github.com/centreon/centreon/blob/develop/centreon/packages/js-config/cypress/e2e/tasks.ts#L154
you can create a cypress command in js-config package which can be inspired by `getContainerIpAddress`

Reply:
I approve because it's a backport and it's already merged on develop
but IMHO, "it doesn't work" without logs or explaination is not a valid answer :) because I'm pretty much sure it can work